Sources/Extensions/DateUtilities.swift

```diff
@@ -66,7 +66,9 @@ internal extension Calendar {
     /// All calculations are done using a gregorian calendar with the UTC timezone
     static let gregorianUTC: Calendar = {
         var cal = Calendar(identifier: .gregorian)
-        cal.timeZone = TimeZone(identifier: "UTC")!
+        if let utc = TimeZone(identifier: "UTC") {
```

Comment:
maybe UTC can be a global constant. I'd rather the library fail (w/ force unwrap) than UTC to not exist for some reason. as a global constant, the failure would happen immediately instead of on-use